### PR TITLE
chore: introduce new marks

### PR DESCRIPTION
**Pull request by glitchminer**
Added new marks

```
    critical: Critical checks for every PR
    settings_all: All tests related to the settings
    settings_keycard: Tests related to Keycard settings settings
    settings_messaging: Tests related to messaging settings
    settings_profile: Tests related to profile settings
    settings_wallet: Tests related to wallet settings
    messaging: All tests related to 1x1 chat and group chat functionality
    communities: All tests related to communities functionality
    onboarding: All tests related to onboarding
    keycard: All tests related to Keycard functionality
    wallet: All tests related to wallet functionality
  ```

### CODE REVIEW

File: tests/onboarding/test_onboarding_generate_new_keys.py [hunk 1]
```diff
@@ -79,8 +80,8 @@ def test_generate_new_keys(main_window, keys_screen, user_name: str, password, u
             assert BiometricsView().is_touch_id_button_visible(), f"TouchID button is not found"
             BiometricsView().wait_until_appears().prefer_password()
         SplashScreen().wait_until_appears().wait_until_hidden()
-        if not configs.system.TEST_MODE:
-            BetaConsentPopup().confirm()
+        # if not configs.system.TEST_MODE:
```
Comment (Valentina1133) [inline tests/onboarding/test_onboarding_generate_new_keys.py:83]: Was it supposed to be commented in PR?

File: tests/wallet_main_screen/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import pytest
+
+marks = pytest.mark.wallet
```
Comment (Valentina1133) [inline tests/wallet_main_screen/__init__.py:3]: Extra line needed at the end of file

File: tests/onboarding/test_onboarding_generate_new_keys.py (hunk 1 above)
Comment (glitchminer) [reply tests/onboarding/test_onboarding_generate_new_keys.py]: Fixed

File: tests/wallet_main_screen/__init__.py (hunk 2 above)
Comment (glitchminer) [reply tests/wallet_main_screen/__init__.py]: Fixed

**Review by Valentina1133 — APPROVED** [review]

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**